### ISSUE TITLE
Where to get example data

### ISSUE THREAD

**Issue by Krasner** [khoadoan/GraphOTSim#4]
In the example it is using PTC_ged dataset (`data/PTC_ged/`). Where is that located?
From the code it seems like these should be json files with a 'ged' field.

**Comment by Krasner** [comment]
Mainly, I want to get my data into the correct format that this code can ingest, so an example json file would be helpful.

**Comment by Krasner** [comment]
Also how is batching actually used? The collate function is never called https://github.com/khoadoan/GraphOTSim/blob/e92bec19ce85347054d01a209c864ed92d14b88c/python/train.py#L346

Additionally in all the similarity computations `torch.mm` is used rather than `torch.bmm`...
It seems like this code would work for batch size = 1, but would mix graph pairs if batch size > 1

**Comment by XMYMouyiXu** [comment]
I have the same problem. Could you please upload a link of the datasets?

**Comment by Krasner** [comment]
RSP, I never got a response but I think all their datasets are standard in pytorch geometric: https://pytorch-geometric.readthedocs.io/en/latest/modules/datasets.html# under `GEDDataset`

Also, LAPJV is called 1-by-1 on each graph pair in the batch - it is not a parallelized operation:
https://github.com/khoadoan/GraphOTSim/blob/e92bec19ce85347054d01a209c864ed92d14b88c/python/models.py#L638

**Comment by Krasner** [comment]
This library used DGL but it can be easily converted to PyTorch Geometric

**Comment by sara-silvaad** [comment]
Hello! Have you figured this out? Even if it;s the GEDDataset it's not clear to me what the .json files have to contain, and it's not clear to me whether the labels are used for anything in training or not. I've been trying to figure it out following the code but I'm afraid I'm missing something
